monocle/worker.py

```diff
@@ -831,7 +831,52 @@ def travel_speed(self, point):
                         pokestop = self.normalize_pokestop(fort)
                         db_proc.add(pokestop)
                 elif fort not in FORT_CACHE:
-                    db_proc.add(self.normalize_gym(fort))
+                    request = self.api.create_request()
+                    request.gym_get_info(
+                                            gym_id=fort.id,
+                                            player_lat_degrees = self.location[0],
+                                            player_lng_degrees = self.location[1],
+                                            gym_lat_degrees=fort.latitude,
+                                            gym_lng_degrees=fort.longitude
+                                        )
```

Comment:
Question, is there some documentation on this method somewhere? I have been successfully calling:
    request.gym_get_info(gym_id=fort.id)
Without the lat/long parameters and it was working fine. I'm not doubting you it means I was probably doing it wrong, but is there some documentation I can see where you see the way to do this?